src/gtc/gtcpp/gtcpp.py

```diff
@@ -227,8 +227,7 @@ def __init__(self, *args: Any, **kwargs: Any):
 
 
 class FieldDecl(ApiParamDecl):
-    # TODO dimensions
-    pass
+    dimensions: Tuple[bool, bool, bool]
```

Comment:
Since the meaning is not directly obvious, how about a small doc string stating each element specifies if the dimension is masked?

Reply:
I had to change this, it might be more obvious in the next commit.